tap_salesforce/salesforce/__init__.py

```diff
@@ -111,6 +112,12 @@
                                            'FlowVariableView',
                                            'AppTabMember',
                                            'ColorDefinition',
+                                           'DatacloudDandBCompany', # Not filterable without a criteria.
```

Comment:
>  Not filterable without a criteria

Is the comment Correct here?

Reply:
This is an error message I have added over here. General comment is already written on top of this list